cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -1398,36 +1376,38 @@ public void testReconcile(Params params, VertxTestContext context) {
         ops.reconcileAll("test", kafkaNamespace, context.succeeding(v -> completeTest.flag()));
     }
 
+
```

Comment:
Nit, but do we need the double empty line?

Reply:
Fixed :)